Automattic/wp-calypso
# Signup: Remove the survey step from the get-dot-blog flow

This pull request removes the survey step from the get-dot-blog signup flow. Fixes https://github.com/Automattic/wp-calypso/issues/9020.
   
#### Testing instructions
  
1. Run `git checkout update/get-dot-blog-flow` and start your server, or open a [live branch](https://delphin.live/?branch=update/get-dot-blog-flow)
2. Open the [`Get.blog signup` page](http://calypso.localhost:3000/start/get-dot-blog?domain=scruffiantest.net)
3. Check that you see the theme step, not the survey step.
    
#### Reviews
  
   
@Automattic/sdev-feed
